github/src/main/kotlin/gropius/sync/github/IMSConfigManager.kt

```diff
@@ -0,0 +1,156 @@
+package gropius.sync.github
+
+import com.lectra.koson.arr
+import com.lectra.koson.obj
+import gropius.model.template.BaseTemplate
+import gropius.model.template.IMSIssueTemplate
+import gropius.model.template.IMSProjectTemplate
+import gropius.model.template.IMSTemplate
+import kotlinx.coroutines.flow.filter
+import kotlinx.coroutines.flow.toSet
+import kotlinx.coroutines.reactor.awaitSingle
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.data.neo4j.core.ReactiveNeo4jOperations
+import org.springframework.data.neo4j.core.findAll
+import org.springframework.stereotype.Component
+
+@Component
+/**
+ * @param neoOperations Reference for the spring instance of ReactiveNeo4jOperations
+ */
+class IMSConfigManager(
+    @Qualifier("graphglueNeo4jOperations")
+    private val neoOperations: ReactiveNeo4jOperations,
+) {
+    /**
+     * JSON schema version to include
+     */
+    private val schema = "https://json-schema.org/draft/2020-12/schema"
```

Comment:
this looks like a const val to me, you can put this at top level or in a companion object (better for scoping)

Reply:
actually there are a lot of constants

Reply:
97e7d2bad00bc5df055901a6122ebc81ba380105